rest/changes_api.go

```diff
@@ -402,14 +402,6 @@ func (h *handler) sendSimpleChanges(channels base.Set, options db.ChangesOptions
 			}
 		}
 
```

Comment:
This change seems unrelated to this enhancement, if this kind of cleanup is justified I think it should be as a standalone PR.

Reply:
It's not unrelated exactly, because I didn't implement deprecated (from go 1.7) CloseNotifier interface in the ResponseWriters.

I can pull it out into a separate commit though.